### ISSUE TITLE
Queue decorator has require params. 

### ISSUE THREAD

**Issue by klerick** [fwoelffel/nest-bull#70]
```
export interface QueueDecoratorOptions {
  name?: string; // Name of the queue
}
```

But if use it without params we get error and wrong interface signature.

https://github.com/fwoelffel/nest-bull/blob/dev/lib/bull.explorer.ts#L30

**Comment by fwoelffel** [comment]
Hi @klerick 
Could you provide some code to reproduce the bug you encounter?

**Comment by klerick** [comment]
I can not do it fast. 
But it's easy to understand here
https://github.com/fwoelffel/nest-bull/blob/dev/lib/bull.explorer.ts#L30

**Comment by fwoelffel** [comment]
Ok I understand your concern. However, if you look at https://github.com/fwoelffel/nest-bull/blob/dev/lib/bull.explorer.ts#L98 and https://github.com/fwoelffel/nest-bull/blob/dev/lib/bull.utils.ts#L1 you'll understand that `BullExplorer.getClassMetadata(metatype).queueName` being `undefined` is not an issue at all.

I'll wait for a few days before closing this issue and I'll take a closer look at it if you find the time to provide some reproducibility code or if someone else can confirm.

**Comment by klerick** [comment]
Firstly, yes, there is a value used by default. However if the name of a queue is set during the module configuration, then we have an error for the default value. I assume that it needs to return a more detailed error of what is going on. For example, 

```
private static getBullQueue(moduleRef: ModuleRef, queueName?: string): Queue {
    try {
      return moduleRef.get<Queue>(getQueueToken(queueName));      
    } catch (e) {
        console.log("Queue 'queueName' not found")
      throw e
    }
  
}
```
or add details to the documentation.
For me the best solution is to set the name of the queue during the module configuration as required.

Secondly, a decorator `@Queue` has an interface
```
export interface QueueDecoratorOptions {
  name?: string; // Name of the queue
}
```
but 
```
BullExplorer.getClassMetadata(metatype).queueName
```
tries to get a property `queueName` instead of `name`. As a result `queueName` is undefined and error returns.

**Comment by fwoelffel** [comment]
Damn! I'm so dumb. Yeah you're right. I'll fix this tonight. I'm sorry for the trouble.

**Comment by klerick** [comment]
it happens. All that matters is that we’ve figured it out :)

**Comment by fwoelffel** [comment]
New release published as 0.6.2
